--- starship-0.48.0/Cargo.toml	2021-01-02T20:42:20+00:00
+++ starship-0.48.0/Cargo.toml	2021-03-01T18:35:48.564180+00:00
@@ -69,10 +69,6 @@
 [dependencies.log]
 version = "0.4.11"
 features = ["std"]
-
-[dependencies.native-tls]
-version = "0.2.7"
-optional = true
 
 [dependencies.notify-rust]
 version = "4.0.0"
@@ -157,12 +153,8 @@
 [features]
 default = ["battery", "http"]
 http = ["attohttpc"]
-tls-vendored = ["native-tls/vendored"]
 [target."cfg(not(windows))".dependencies.nix]
-version = "0.19.1"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["winuser", "securitybaseapi", "processthreadsapi", "handleapi", "impl-default"]
+version = "0.20.0"
 [badges.is-it-maintained-issue-resolution]
 repository = "starship/starship"
 
