--- starship-0.41.3/Cargo.toml	2020-05-15T15:10:30+00:00
+++ starship-0.41.3/Cargo.toml	2020-05-16T17:47:49.948287+00:00
@@ -35,7 +35,7 @@ path = "src/main.rs"
 version = "0.12.1"
 
 [dependencies.attohttpc]
-version = "0.14.0"
+version = "0.15.0"
 features = ["tls", "form"]
 optional = true
 default-features = false
@@ -140,7 +140,6 @@ version = "3.1.0"
 [features]
 default = ["battery", "http"]
 http = ["attohttpc"]
-tls-vendored = ["native-tls/vendored"]
 [badges.is-it-maintained-issue-resolution]
 repository = "starship/starship"
 
