--- starship-1.2.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ starship-1.2.1/Cargo.toml	2022-07-22T12:01:20.949681+00:00
@@ -55,7 +55,7 @@
 version = "0.2.1"
 
 [dependencies.git2]
-version = "0.13.25"
+version = "0.13.20"
 default-features = false
 
 [dependencies.indexmap]
@@ -74,7 +74,7 @@
 version = "1.9.0"
 
 [dependencies.open]
-version = "2.0.2"
+version = "3.0.2"
 
 [dependencies.os_info]
 version = "3.0.9"
@@ -179,9 +179,6 @@
 default = ["battery"]
 [target."cfg(not(windows))".dependencies.nix]
 version = "0.23.1"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["winuser", "securitybaseapi", "processthreadsapi", "handleapi", "impl-default"]
 [badges.is-it-maintained-issue-resolution]
 repository = "starship/starship"
 
