--- starship-0.56.0/Cargo.toml	1970-01-01T00:00:00+00:00
+++ starship-0.56.0/Cargo.toml	2021-08-01T22:44:27.237226+00:00
@@ -65,16 +65,12 @@
 default-features = false
 
 [dependencies.indexmap]
-version = "1.7.0"
+version = "1.6.2"
 features = ["serde"]
 
 [dependencies.log]
 version = "0.4.14"
 features = ["std"]
-
-[dependencies.native-tls]
-version = "0.2.7"
-optional = true
 
 [dependencies.notify-rust]
 version = "4.5.1"
@@ -166,8 +162,6 @@
 
 [dependencies.yaml-rust]
 version = "0.4.5"
-[dev-dependencies.mockall]
-version = "0.10"
 
 [dev-dependencies.tempfile]
 version = "3.2.0"
@@ -177,12 +171,8 @@
 [features]
 default = ["battery", "http"]
 http = ["attohttpc"]
-tls-vendored = ["native-tls/vendored"]
 [target."cfg(not(windows))".dependencies.nix]
-version = "0.21.0"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["winuser", "securitybaseapi", "processthreadsapi", "handleapi", "impl-default"]
+version = "0.22.0"
 [badges.is-it-maintained-issue-resolution]
 repository = "starship/starship"
 
