--- starship-1.2.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ starship-1.2.1/Cargo.toml	2022-02-05T11:09:48.527381+00:00
@@ -55,11 +55,11 @@
 version = "0.2.1"
 
 [dependencies.git2]
-version = "0.13.25"
+version = "0.13.20"
 default-features = false
 
 [dependencies.indexmap]
-version = "1.8.0"
+version = "1.7.0"
 features = ["serde"]
 
 [dependencies.log]
@@ -118,7 +118,7 @@
 version = "1.0.74"
 
 [dependencies.sha-1]
-version = "0.10.0"
+version = "0.9.8"
 
 [dependencies.shadow-rs]
 version = "0.8.1"
@@ -167,7 +167,7 @@
 [dependencies.yaml-rust]
 version = "0.4.5"
 [dev-dependencies.mockall]
-version = "0.11"
+version = "0.10"
 
 [dev-dependencies.tempfile]
 version = "3.2.0"
@@ -179,9 +179,6 @@
 default = ["battery"]
 [target."cfg(not(windows))".dependencies.nix]
 version = "0.23.1"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["winuser", "securitybaseapi", "processthreadsapi", "handleapi", "impl-default"]
 [badges.is-it-maintained-issue-resolution]
 repository = "starship/starship"
 
